### DISCUSSION TITLE
Why was the Scylla plugin to create dump files removed from the distribution?

### DISCUSSION THREAD

**Discussion by FransBouma** [x64dbg/x64dbg#3446] [Q&A] [answered]
I use x64dbg regularly and for snapshotting exe's (I do game engine reverse engineering) I use the Scylla plugin to create a dump file of a running exe. I was using a x64dbg version from January 2023 and thought "let's update to the latest build", but to my surprise the Scylla plugin to create a dump file is no longer there (or I can't find it). 

Was this feature removed for something else, or is it moved to another menu and am I overlooking it? I now use a build from July last year which seemed to be one of the last builds still containing it. 

If there are other ways to reliably create a dump file like Scylla did, I'm all ears too. TIA!

**Comment by holysnipz** [comment, marked as answer]
Scylla is still included, but [no longer under Plugins menu](https://github.com/x64dbg/x64dbg/commit/87035561491281d002516a25e3894990305301f7).

Instead use this button or using command line below
![image](https://github.com/user-attachments/assets/20aa51a8-d7a3-4624-960e-0569786a0221)

or via hotkey
![image](https://github.com/user-attachments/assets/7fb043f3-8970-49c1-9b09-b958a71dbe77)

**Comment by mrexodia** [comment]
People constantly confuse Scylla with ScyllaHide and this often leads to frustrating discussions. Eventually I will remove the Scylla plugin from the snapshot completely, because it’s been unmaintained for years. You will still be able to install it manually though (or via a plugin manager).

**Comment by FransBouma** [comment]
Thanks @mrexodia and @holysnipz !
